webapp/portlet/src/main/webapp/vue-apps/administration-navigation/components/ExoAdministrationHamburgerNavigation.vue

```diff
@@ -148,8 +149,13 @@ export default {
         vuetify: Vue.prototype.vuetifyOptions,
       }).$mount(parentId);
     },
-    openDrawer() {
-      this.$emit('open-second-level');
+    toggleOpenDrawer() {
+      this.secondeLevel = !this.secondeLevel;
+      if (this.secondeLevel) {
+        this.$emit('open-second-level');
+      } else {
+        this.$emit('close-second-level');
+      }
```

Comment:
Just for code improvement @SaraBoutej @AzmiTouil . It would be better to do it this way  : 
```javascript
  watch: {
...
   secondeLevel() {
      if (this.secondeLevel) {
        this.$emit('open-second-level');
      } else {
        this.$emit('close-second-level');
      }
   },
...
  },
...
  methods: {
...
    toggleOpenDrawer() {
      this.secondeLevel = !this.secondeLevel;
    },
...
```
This way, in a centralized way, when changing the value of `secondeLevel`, you will always emit the dedicated events related to exact value of `secondeLevel`. ( advice for future improvements for better code maintainability & low complexity )